pysages/collective_variables/shape.py

```diff
@@ -104,7 +104,7 @@ def function(self):
         Returns
         -------
         Callable
-            Function to calculate the Eigenvalue with the specified axis index \
+            Function to calculate the eigenvalue with the specified axis index \
             of the GyrationTensor. \
```

Comment:
--> gyration_tensor?

Reply:
https://en.wikipedia.org/wiki/Gyration_tensor

It has a wikipedia page. Is that googlable enough, or should we write down the equations.

Reply:
Sorry, my suggestion wasn't clear. I just meant whether we should replace "GyrationTensor" with "gyration_tensor" (the actual name of the function) or actually simply with "gyration tensor". Eventually, I think it should just be "gyration tensor", so I went for that (6e76c1d651812504aa8a9aad7f01b8d0201d177d). If that's OK, go ahead and merge, otherwise I'll change it.

I think there's no need to write down the equations.